Python/check_in.py

```diff
@@ -34,14 +34,7 @@ def commit_data(username, password):
 def get_hour_message():
     h = datetime.datetime.fromtimestamp(
         int(time.time()), pytz.timezone('Asia/Shanghai')).hour
-    if 6 <= h <= 11:
-        return '晨'
-    elif 12 <= h <= 17:
-        return '午'
-    elif 18 <= h <= 24:
-        return '晚'
-    else:
-        return '凌晨'
+    return "睡晨午晚"[h // 6]
```

Comment:
可以用 list 来保持原有逻辑，不过有没有可能 `h >=24` ？

```suggestion
    return ['凌晨', '晨', '午', '晚'][h // 6]
```

另外这些时间节点是哪里定义的？需要传给其他 API 吗，还是仅供用户界面显示？